webrtc.html

```diff
@@ -6042,7 +6042,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <p>The <dfn><code>stop</code></dfn> method stops the
               <a><code>RTCRtpTransceiver</code></a>. The sender of this
               transceiver will no longer send, the receiver will no longer
-              receive, and the <a>negotiation-needed flag</a> is set.</p>
+              receive, <code>receiver.track</code>'s "onended" event will
+              fire, and the <a>negotiation-needed flag</a> is set.</p>
```

Comment:
I guess the firing or not of the ended event can be discussed. While it's clear we should not fire if the app does ```stop()``` on the track, it is a secondary effect here coming from stopping the transeiver.